src/main/kotlin/io/perfometer/http/Http.kt

```diff
@@ -49,7 +51,13 @@ data class HttpRequest(val method: HttpMethod,
                        val body: ByteArray,
                        val consumer: (HttpResponse) -> Unit)
 
-class HttpResponse(val status: HttpStatus, val body: ByteArray = byteArrayOf()) {
+class HttpResponse(val status: HttpStatus, val headers: Map<String, String>, val body: ByteArray = byteArrayOf()) {
 
-    fun asString(): String = body.toString(Charsets.UTF_8)
+    private fun charsetFromContentType(): Charset {
```

Comment:
Cosider extracting this function to a separate file, and testing it separately - may be useful in other contexts too.